src/accessor/util.rs

```diff
@@ -8,8 +8,8 @@ fn buffer_view_slice<'a, 's>(
     view: buffer::View<'a>,
     get_buffer_data: &dyn Fn(buffer::Buffer<'a>) -> Option<&'s [u8]>,
 ) -> Option<&'s [u8]> {
-    let start = view.offset();
-    let end = start + view.length();
+    let start = usize::try_from(view.offset()).ok()?;
+    let end = usize::try_from(start as u64 + view.length()).ok()?;
```

Comment:
Isn't this going to fail on 32bit systems?

Reply:
Yes, this will fail on 32bit systems when the buffer is too large to fit into the 32bit address space, which seems fair to me. I guess technically the part of the buffer to be accessed in this function specifically may fit, but I'm not sure that edge case is worth handling. Would you like to handle it differently?